backend/app/test/userTest.ts

```diff
@@ -0,0 +1,117 @@
+import chai = require('chai')
+import chaiHttp = require('chai-http')
+import assert = require('assert')
+import { v4 as uuidv4 } from 'uuid'
+
+chai.use(chaiHttp)
+
+describe('Testing the login/register', () => {
```

Comment:
You test both so it's not login/register

```suggestion
describe('Testing the login and register', () => {
```